### PR TITLE
Style guide updates

### PR DESCRIPTION
**Pull request by Mats-SX**

### CODE REVIEW

File: docs/style-guide.adoc [hunk 1]
```diff
@@ -7,12 +7,13 @@
 toc::[]
 
 This is the style guide for the Cypher language, in the context of its standardization through the openCypher project.
-This document contains rules for how to use the language, intended to help construct a consistent and portable usage of the language across many use cases and implementations.
-There are also more peripheral recommendations, which are to be taken more lightly, and reflects attempts at helping users of the language to treat language-related concerns in a consistent manner.
+This document consists of two main parts: <<Rules>> and <<Recommendations>>.
```
Comment (petraselmer) [inline docs/style-guide.adoc]: "parts" -> "sections"

File: docs/style-guide.adoc [hunk 2]
```diff
@@ -7,12 +7,13 @@
...
 
-NOTE: Cypher is a language.
-Languages have their own rules, and are not eager to adapt to circumstance.
-Remember not to be too eager to shiv a Cypher expression into the conventions of a different language.
+In <<Rules>>, we list syntax guidelines for composing Cypher queries in a conventional, readability-oriented way.
+The examples provided always transform _valid_, but poorly formatted, queries into a different query in the conventional format, with the exact same semantics.
```
Comment (petraselmer) [inline docs/style-guide.adoc]: "with the exact same semantics" -> "whilst retaining the same semantics"

Comment (petraselmer) [inline docs/style-guide.adoc]: "conventional" -> "recommended"

File: docs/style-guide.adoc [hunk 3]
```diff
@@ -7,12 +7,13 @@
...
+
+In <<Recommendations>>, we list guidelines may have an effect on the semantics of queries, such as the way a graph schema is composed through label and relationship types.
```
Comment (petraselmer) [inline docs/style-guide.adoc]: "guidelines may have" -> "guidelines that may have"

File: docs/style-guide.adoc [hunk 4]
```diff
@@ -7,12 +7,13 @@
...
+Bear in mind that these recommendations will not work after-the-fact: if the graph has been constructed with one set of label and relationship types, queries in the associated workload can not be re-formatted according to these recommendations without also refactoring the data graph.
```
Comment (petraselmer) [inline docs/style-guide.adoc]: "can not" -> "cannot"

File: docs/style-guide.adoc [hunk 5]
```diff
@@ -41,6 +42,10 @@ RETURN otherGroup.name, collect(topic.name), sum(score) AS score
 ORDER BY score DESC
 ----
 
+---
+
+The purpose of this document is to help users of the language to share queries with each other with minimal friction, and to construct a consistent and portable usage of the lanugage across many use cases and implementations.
```
Comment (petraselmer) [inline docs/style-guide.adoc]: "lanugage" -> "language"

File: docs/style-guide.adoc (hunk 4 above)
Comment (Mats-SX) [inline docs/style-guide.adoc]: I seem to never learn this one! :p

**Comment by Mats-SX** [comment]
@petraselmer Thanks for the review, updated.

**Pull request merged**